### PR TITLE
Feature news page

### PR DESCRIPTION
**Pull request by bejanaf**
Hi to all developers, I finished my ✨ News Page ✨ ! I would be very happy if someone could take a look at my code.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +0,0 @@
-API_KEY=910c6caaee6945439c19ae099321f342
```
Comment (SabMueller) [inline .env:1]: ohh interesting... :D

**Review by SabMueller — APPROVED** [review]
Looks good to me! But what happened with your .env file..? ✨ :D

File: .env (hunk 1 above)
Comment (farahccino) [reply .env:1]: lol

**Comment by bejanaf** [comment]
After my first push, the file was already online. Big mistake! The file has now been deleted, and I have already contacted the provider about this event and already using a new key ;)

**Pull request merged**